resources/js/components/ui/InfoIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import {A, AnchorProps} from "@solidjs/router";
+import {ImInfo} from "solid-icons/im";
+import {Match, Switch, VoidComponent} from "solid-js";
+import {htmlAttributes, useLangFunc} from "../utils";
+import {Button} from "./Button";
+
+interface ButtonProps extends htmlAttributes.button {
```

Comment:
htmlAttributes.button powinno być zapisane jako HtmlAttributes.Button

typy i namespace'y w TSie powinny być z wielkiej litery, ale to tylko konwencja

Reply:
Namespace może być z dużej albo małej, są różne konwencje. A typy są owszem z dużej ale tutaj zrobiłem z małej bo to są propsy natywnego `<button>` więc jest `button` bo wygląda podobnie. Trzymanie się tutaj konwencji wydaje mi się podobnie sztywniackie co prettier :) Zresztą to już od dosyć dawna jest.